packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -457,6 +457,7 @@ describe('Table', () => {
             const originalRenderedRowCount = pageObject.getRenderedRowCount();
 
             element.style.height = '700px';
+            element.viewport.scrollTop = 2;
```

Comment:
@msmithNI After these dependency updates I find that this test fails reliably on CI and locally unless I kick the virtualizer here by updating scrollTop here. I'm not sure what's going on, but the sizing of things definitely looks wrong before kicking it.

When I break above right after the waitForUpdatesAsync I can see the virutalizer has created way too many elements:
![image](https://user-images.githubusercontent.com/1588923/215227539-61c6bcb9-29c7-46a0-a832-bc11a794c0e0.png)

If while still there I jiggle the scrollbar you can see it figures it out:
![image](https://user-images.githubusercontent.com/1588923/215227666-772199a5-6f64-4592-8cd3-611e0a54830e.png)

Reply:
Created https://github.com/ni/nimble/issues/1008